sirepo/package_data/static/js/sirepo-plotting.js

```diff
@@ -4088,21 +4099,32 @@ SIREPO.app.directive('parameterPlot', function(appState, focusPointService, layo
                 });
                 updateYLabel();
                 plots.forEach(function(plot, i) {
-                    if (cachedPlotVisibilty(i, $scope.modelName)) {
+                    if (cachedPlotVisibility(i, $scope.modelName)) {
                         setPlotVisible(i, ! isPlotVisible(i));
                     }
                 });
 
-                $scope.$on(
-                    $scope.modelName + '.changed',
-                    () => {
+                function handlePlotSelection() {
+                    const c = $scope.plotLabels(plots);
+                    const p = appState.models[$scope.modelName].selectedPlots || c;
+                    if (JSON.stringify(c) != JSON.stringify(p)) {
```

Comment:
Labels are not necessarily unique so this may not be a valid equality test. Also use `===` wherever possible as a matter of convention, though it doesn't really matter here...

Reply:
not unique in the sense that two different y values might have the same label? If so, when would that happen?

Reply:
One would hope that the labels are always different, but they're arbitrary. This change affects all parameter plots, so better to be cautious I think

Reply:
Ok, maybe I could use 'field' instead of 'label'. I think i'll have to make some schema changes

Reply:
I guess I don't know what plot changes this is supposed to pick up; maybe wait to see what @moellep has to say?